rancher/v1.5/en/cattle/webhook-service/index.md

```diff
@@ -1,103 +1,49 @@
----
-title: Webhooks in Rancher
-layout: rancher-default-v1.5
-version: v1.5
-lang: en
----
```

Comment:
This front matter is required for the docs site to render correctly in GH pages. Please do not delete it.